openmdao/vectors/vector.py

```diff
@@ -491,6 +491,19 @@ def set_const(self, val):
         """
         pass
 
+    def dot(self, vec):
+        """
+        Compute the dot product of the current vec and the incoming vec.
+
+        Must be implemented by the subclass.
+
+        Parameters
+        ----------
+        vec : <Vector>
+            the vector whose values self is set to.
```

Comment:
It looks like this comment may have been copied from `set_vec` without being updated.